boto/utils.py

```diff
@@ -104,6 +104,33 @@ def unquote_v(nv):
         return (nv[0], urllib.parse.unquote(nv[1]))
 
 
+def parse_qs_safe(qs, keep_blank_values=False, strict_parsing=False,
+                  encoding='utf-8', errors='replace'):
```

Comment:
Ah, I see, compatibility with the default values for `parse_qs`.  Seems reasonable.